### PR TITLE
FPL-96: Add hint text when adding case title

### PR DESCRIPTION
**Pull request by Jamesnelson117**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FPL-96

### Change description ###
Added the ability to add case name on opening of a new case
Change case name functionality
Tests for adding / editing case name
 
screen shot 2018-11-06 at 13 18 49

screen shot 2018-11-06 at 13 19 38

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/pages/createCase/createCase.js [hunk 1]
```diff
@@ -6,13 +6,24 @@ module.exports = {
...
+    enterCaseName: {
+      caseName: '#caseName',
+    },
   },
+  testCaseName: 'Barnet Council v Smith',
+  continueButton: 'Continue',  
   startButton: 'Start',
 
+  enterCaseName() {
+    I.fillField(this.fields.enterCaseName.caseName, this.testCaseName);
```
Comment (kurt-uk) [inline e2e/pages/createCase/createCase.js]: i dont know if the structure here should be like this, so do you need `enterCaseName` inside fields? could you just have `Fields > caseName`?

File: e2e/pages/changeCaseName/changeCaseName.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    caseName: '#caseName',
+  },
+
+  changeCaseName(caseName='Craigavon council v Smith') {
```
Comment (kurt-uk) [inline e2e/pages/changeCaseName/changeCaseName.js]:
```suggestion
  changeCaseName(caseName = 'Craigavon council v Smith') {
```

**Comment by sivas23** [comment]
QA - Pass

File: ccd-definition/CaseEventToFields.json [hunk 3]
```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "CaseEventID": "openCase",
+    "CaseFieldID": "caseName",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "OPTIONAL",
+    "PageID": 1,
+    "PageLabel": "",
```
Comment (geek-soft) [inline ccd-definition/CaseEventToFields.json]: Please remove if not used.

File: e2e/pages/createCase/createCase.js [hunk 4]
```diff
@@ -7,12 +7,23 @@ module.exports = {
...
     event: 'event',
   },
+  enterCaseNamePage: {
+    caseName: '#caseName',
+  },
+  testCaseName: 'Barnet Council v Smith',
+  continueButton: 'Continue',  
   startButton: 'Start',
 
+  enterCaseName() {
```
Comment (geek-soft) [inline e2e/pages/createCase/createCase.js]: Please introduce argument to the method that takes case name.
```suggestion
  enterCaseName(caseName = 'Barnet Council v Smith') {
```

File: e2e/pages/createCase/createCase.js [hunk 5]
```diff
@@ -7,12 +7,23 @@ module.exports = {
     caseType: 'case-type',
     event: 'event',
   },
+  enterCaseNamePage: {
+    caseName: '#caseName',
+  },
+  testCaseName: 'Barnet Council v Smith',
+  continueButton: 'Continue',
```
Comment (geek-soft) [inline e2e/pages/createCase/createCase.js]: Can we put continue button underneath start button?

File: e2e/paths/loginAsHmctsAdmin_test.js [hunk 6]
```diff
@@ -23,3 +23,8 @@ xScenario('HMCTS admin can login and add a FamilyMan case number to a submitted
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.addFamilyManCaseNumber);
 });
+
+xScenario('HMCTS admin can login and see case name', (I, loginPage, createCasePage) => {
```
Comment (geek-soft) [inline e2e/paths/loginAsHmctsAdmin_test.js]: The problem with the test is that it relies on data entered by other tests which makes it fragile. I suggest removing it at this stage.

**Pull request merged**